curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-087.md

```diff
@@ -0,0 +1,190 @@
+---
+id: 5f46e270702a8456a664f0df
+title: Part 87
+challengeType: 0
+isHidden: true
+---
+
+## Description
+<section id='description'>
+
+To remove some of the vertical space between the `h1` element and the text `Est. 2020`, change the bottom margin of the `h1` to `15px`;
```

Comment:
To remove some of the vertical space between the `h1` element and the text `Est. 2020`, change the bottom margin of the `h1` to `15px`.